src/PKSim.Presentation/Views/Simulations/IPopulationPKAnalysisView.cs

```diff
@@ -7,9 +7,9 @@ namespace PKSim.Presentation.Views.Simulations
 {
    public interface IPopulationPKAnalysisView : IView<IPopulationPKAnalysisPresenter>
    {
-      void BindTo(PKAnalysisDTO dataTable);
+      void BindTo(PKAnalysisDTO pkAnalysisOnCurveDTO, PKAnalysisDTO pkAnalysisOnIndividualsDTO);
```

Comment:
WE typicallyh only bind to ONE object. This is the DTO you create to bind to the view